modules/index.js

```diff
@@ -1,6 +1,9 @@
-import UserInterFace  from './userInterface.js';
+// Using Class Book represents a book and the various values used
```

Comment:
- [ ] Good job so far using your modules folder, however, for this project, it is required that your `index.js` file be outside of the `modules` folder, so you can let your `index.js` file in the root directory of your project, and use the `modules` folder for some dependencies for the `index.js`